applications/GeoMechanicsApplication/tests/cpp_tests/test_apply_c_phi_reduction_process.cpp

```diff
@@ -0,0 +1,105 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Anne van de Graaf
+//
+#include "containers/model.h"
+#include "custom_processes/apply_c_phi_reduction_process.hpp"
+#include "custom_constitutive/linear_elastic_plane_strain_2D_law.h"
+#include "geometries/quadrilateral_2d_4.h"
+#include "processes/structured_mesh_generator_process.h"
+#include "stub_linear_elastic_law.h"
+#include "testing/testing.h"
+#include <boost/numeric/ublas/assignment.hpp>
+
+#include <boost/algorithm/cxx11/all_of.hpp>
+#include <boost/algorithm/cxx11/none_of.hpp>
```

Comment:
I don't think these includes are necessary, in any case there are standard versions of these (i.e. std::all_of, std::none_of

Reply:
removed